site/components/Wallet.js

```diff
@@ -17,15 +17,8 @@ const removeLastConnector = () =>
   window.localStorage.removeItem(persistLastConnectorKey)
 
 const Wallet = function () {
-  const {
-    account,
-    active,
-    activate,
-    connector,
-    deactivate,
-    error,
-    setError
-  } = useWeb3React()
+  const { account, active, activate, connector, deactivate, error, setError } =
+    useWeb3React()
```

Comment:
Gotta say, this looks far weirder than before 😅

Reply:
Prettier 🤷🏼